bgfx.idl

```diff
@@ -1546,8 +1558,11 @@ func.setViewClear
 	.flags   "uint16_t" --- Clear flags. Use `BGFX_CLEAR_NONE` to remove any clear
 	                    --- operation. See: `BGFX_CLEAR_*`.
 	.rgba    "uint32_t" --- Color clear value.
+	 { default = 0x000000ff }
```

Comment:
You may want to use `"0x000000ff"` here,  because `0x000000ff` will convert to `255` by generator.

Reply:
Changed.